src/django/api/views/v1/opensearch_query_builder/production_locations_query_builder.py

```diff
@@ -98,3 +98,51 @@ def add_search_after(
         id_type='os_id'
     ):
         super().add_search_after(search_after_value, search_after_id, id_type)
+
+    def add_specific_queries(self, query_params):
```

Comment:
I would suggest thinking about changing or re-naming this method, it basically combines two methods `add_aggregations` and `add_multi_match`, and it's hard to figure out what it does from the name of the method.

Reply:
Changed the approach.
Thank you!